Framework/Core/include/Framework/DataAllocator.h

```diff
@@ -313,20 +314,51 @@ class DataAllocator
                   "pointer to data type not supported by API. Please pass object by reference");
   }
 
+  template <typename T>
+  auto make(OutputRef const& ref)
+  {
+    return make<T>(getOutputByBind(ref));
+  }
+
+  template <typename T>
+  auto make(OutputRef const& ref, size_t n)
+  {
+    return make<T>(getOutputByBind(ref), n);
+  }
+
+  template <typename T, typename... Args>
+  auto make(OutputRef const& ref, Args... args)
```

Comment:
I think this will pass all arguments by copy, doesn't it?

Reply:
fixed.